.github/workflows/testing.yml

```diff
@@ -81,17 +81,17 @@ jobs:
       with:
         fetch-depth: 0
 
+    - name: Install graphviz and pandoc
+      run: sudo apt-get install graphviz pandoc
+
     - name: Set up Python
       uses: actions/setup-python@v4
       with:
         python-version: '3.11'
 
-    - name: Install Python dependencies
+    - name: Install tox
       run: python -m pip install --progress-bar off --upgrade tox
 
-    - name: Install language-pack-fr and tzdata
-      run: sudo apt-get install graphviz pandoc
-
```

Comment:
Looks like we had this step named wrong, which I fixed above to reflect the packages that we're actually installing here.